newIDE/app/src/fixtures/animation-speed-scale/animation-speed-scale.json

```diff
@@ -324,7 +324,7 @@
           "layer": "",
           "locked": false,
           "name": "Player",
-          "persistentUuid": "d9e856da-e964-43e9-9654-29a48ad5d677",
+          "persistentUuid": "77304a89-868a-4b15-99cc-dc44ec10ad3c",
```

Comment:
Note to me: I really have to find a solution to avoid these random UUIDs in the fixtures!